lib/SQLiteReporter.cpp

```diff
@@ -215,6 +206,24 @@ void Mutang::SQLiteReporter::reportResults(const std::unique_ptr<Result> &result
   outs() << "Results can be found at '" << databasePath << "'\n";
 }
 
+#pragma mark -
+
+std::string SQLiteReporter::getCallerPathAsString(std::vector<std::string> callerPath) {
```

Comment:
Xcode denies `push_back()` on both ways. I have made a change with the second form.